plugins/services/src/js/schemas/service-schema/Networking.js

```diff
@@ -11,7 +11,7 @@ const Networking = {
   title: 'Network',
   description: (
     <span>
-      Configure the networking for your service. We will automatically generate a Service Address to connect to for each of your load balanced endpoints. <a href="https://docs.mesosphere.com/1.8/usage/service-discovery/load-balancing-vips/virtual-ip-addresses/" target="_blank">Read more about load balancing</a>.
+      Configure the networking for your service. We will automatically generate a Service Address to connect to for each of your load balanced endpoints. <a href="https://docs.mesosphere.com/usage/service-discovery/load-balancing-vips/virtual-ip-addresses/" target="_blank">Read more about load balancing</a>.
```

Comment:
1. Dynamic docs link
2. break this into multiple lines. You are going to hit errors when this thing lands: https://github.com/dcos/dcos-ui/pull/1866